ocs_ci/templates/workloads/fio/customresource.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: ripsaw.cloudbulldozer.io/v1alpha1
```

Comment:
Please change the file name to more meaningful

Reply:
changed to `fio_cr_template.yaml`